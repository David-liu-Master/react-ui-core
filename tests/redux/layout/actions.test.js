import {
  toggleDrawer
} from '../../../src/redux/layout/actions';

describe('layout actions', () => {
  it('should create toggleDrawer action', () => {
    expect(toggleDrawer()).toMatchSnapshot()
  })
})
