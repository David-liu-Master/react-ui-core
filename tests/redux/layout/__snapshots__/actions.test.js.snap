// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout actions should create toggleDrawer action 1`] = `
Object {
  "type": "openpatch.ui-core.layout.TOGGLE_DRAWER",
}
`;
